crates/verify/src/verify.rs

```diff
@@ -275,13 +275,13 @@ impl VerifyArgs {
 
             let cache = project.read_cache_file().ok();
 
-            let version = if let Some(ref version) = self.compiler_version {
-                version.trim_start_matches('v').parse()?
-            } else if let Some(ref solc) = config.solc {
+            let version = if let Some(ref solc) = config.solc {
                 match solc {
                     SolcReq::Version(version) => version.to_owned(),
                     SolcReq::Local(solc) => Solc::new(solc)?.version,
                 }
+            } else if let Some(ref version) = self.compiler_version {
```

Comment:
cli arg should come first or alternatively it is merged onto config and then just config is present here

Reply:
good point, just realized https://github.com/superform-xyz/SuperVaults/blob/testProdDeployment/script/utils/verify_contracts.sh#L32C1-L32C49 so not sure we actually need any change here. ...